src/interfaces/ISablierV2Lockup.sol

```diff
@@ -290,19 +289,19 @@ interface ISablierV2Lockup is
     /// @param newRecipient The address of the new owner of the stream NFT.
     function withdrawMaxAndTransfer(uint256 streamId, address newRecipient) external;
 
-    /// @notice Withdraws assets from streams to the provided address `to`.
+    /// @notice Withdraws assets from streams to the owner of `streamIds`.
     ///
     /// @dev Emits multiple {Transfer}, {WithdrawFromLockupStream}, and {MetadataUpdate} events.
     ///
     /// Notes:
     /// - This function attempts to call a hook on the recipient of each stream, unless `msg.sender` is the recipient.
     ///
     /// Requirements:
-    /// - All requirements from {withdraw} must be met for each stream.
```

Comment:
The reason I removed that is because the following 2 additional requirements from `{withdraw}` do not hold true for `withdrawMultiple` since it does not `to` parameter now.

- `to` must be the recipient if `msg.sender` is not the stream's recipient or an approved third party.
- `to` must not be the zero address.

Reply:
i know,  my reasoning  is that it calls the `withdraw` underneath so we could still say this, thus i didn't want to repeat ourselves too much

do you say it would be better to say:
```
    /// - Must not be delegate called.
    /// - There must be an equal number of `streamIds` and `amounts`.
    /// - Each stream id in the array must not reference a null or depleted stream.
    /// - Each amount in the array must be greater than zero and must not exceed the withdrawable amount.
```

Reply:
Yes. I like this one. It's more explicit to this function.

Reply:
ok, agree